src/queries/most-active-wallet-within-range.sql.ts

```diff
@@ -1,38 +1,30 @@
 import { sql } from 'slonik';
 import { Params } from './Params';
 
-export default (params: Params) => {
+export default () => {
+  const currentDate = new Date();
+  const today = currentDate.getDate();
+  const dayOfTheWeek = currentDate.getDay();
+  const blockchain_timestamp_precision = 1000000; // miliseconds to nanoseconds
 
-  const conditions = [];
-
-  if (
-    params.after_block_timestamp !== undefined &&
-    params.after_block_timestamp > 0
-  ) {
-    conditions.push(
-      sql`transactions.block_timestamp >= ${params.after_block_timestamp}`,
-    );
-  }
-
-  if (
-    params.before_block_timestamp !== undefined &&
-    params.before_block_timestamp > 0
-  ) {
-    conditions.push(
-      sql`transactions.block_timestamp <= ${params.before_block_timestamp}`,
-    );
-  }
+  const startOfWeek = new Date().setDate(today - (dayOfTheWeek || 7));
+  const startOfWeekTimeStamp = new Date(startOfWeek).getTime() * blockchain_timestamp_precision;
+  const currentDateTimeStamp = currentDate.getTime() * blockchain_timestamp_precision;
```

Comment:
This code makes the query no longer query within a range, but rather forces it to be a week (since it accepts no parameters)

Reply:
I see I'm giving conflicting messages here. I think the name you had at first was correct for this query. I will make the necessary changes & merge.